tests/test_entities.py

```
@@ -1075,6 +1195,76 @@ async def test_switch_entities_created(
+            scheduled_auto_off: Callable[[], None] | None = None
+            auto_off_handle = MagicMock()
+            original_call_later = hass.loop.call_later
+
+            def capture_auto_off(
+                delay: float,
+                callback: Callable[..., None],
+                *args: object,
+            ) -> object:
+                nonlocal scheduled_auto_off
+                if getattr(callback, "__name__", "") == "auto_off_callback":
+                    scheduled_auto_off = lambda: callback(*args)
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `scheduled_auto_off` as a `def`

(E731)